### PR TITLE
Initial SQL scripts for db

### PR DESCRIPTION
**Pull request by gfitzp2002**

### CODE REVIEW

File: 002-Update-Delivery-Employee.sql
Comment (shaunganley) [inline 002-Update-Delivery-Employee.sql:1]: don't think this is needed?

File: SQL/create_delivery_employee.sql [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    employee_id int PRIMARY KEY AUTO_INCREMENT,
+    f_name varchar(255),
+    l_name varchar(255),
+    salary decimal,
+    bank_acc_num varchar(20),
+    ni_num varchar(11)
+);
+
+CREATE TABLE delivery_employee (
+    delivery_employee_id int PRIMARY KEY AUTO_INCREMENT,
```
Comment (shaunganley) [inline SQL/create_delivery_employee.sql]: do you need this field?

File: SQL/create_project_technology_tech_project [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    tech_lead int
+);
+
+CREATE TABLE technology (
+    technology_id int PRIMARY KEY AUTO_INCREMENT,
+    technology_name varchar(50)
+);
+
+CREATE TABLE tech_project (
+    tech_project_id int PRIMARY KEY AUTO_INCREMENT,
```
Comment (shaunganley) [inline SQL/create_project_technology_tech_project]: do you need this field?

File: SQL/create_sales_employee.sql [hunk 3]
```diff
@@ -0,0 +1,6 @@
+CREATE TABLE sales_employee (
+    sales_employee_id int PRIMARY KEY AUTO_INCREMENT,
```
Comment (shaunganley) [inline SQL/create_sales_employee.sql:2]: do you need this field?

File: create_delivery_employee.sql [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    employee_id int PRIMARY KEY AUTO_INCREMENT,
+    f_name varchar(255),
+    l_name varchar(255),
+    salary decimal,
+    bank_acc_num varchar(20),
+    ni_num varchar(11)
+);
+
+CREATE TABLE delivery_employee (
+    delivery_employee_id int PRIMARY KEY AUTO_INCREMENT,
```
Comment (shaunganley) [inline create_delivery_employee.sql]: do you need this field?

**Review by shaunganley — CHANGES_REQUESTED** [review]

File: SQL/create_delivery_employee.sql (hunk 1 above)
Comment (gfitzp2002) [reply SQL/create_delivery_employee.sql]: Removed field

Comment (gfitzp2002) [reply SQL/create_delivery_employee.sql]: removed

File: SQL/create_project_technology_tech_project (hunk 2 above)
Comment (gfitzp2002) [reply SQL/create_project_technology_tech_project]: removed and composite key created

File: SQL/create_sales_employee.sql (hunk 3 above)
Comment (gfitzp2002) [reply SQL/create_sales_employee.sql:2]: removed

File: create_delivery_employee.sql (hunk 4 above)
Comment (gfitzp2002) [reply create_delivery_employee.sql]: removed

**Review by shaunganley — APPROVED** [review]

**Pull request merged**